packages/dbos-cloud/cli.ts

```diff
@@ -365,6 +365,16 @@ orgCommands
     process.exit(exitCode);
   }))
 
+orgCommands
+  .command('rename')
+  .description("Rename the organization")
+  .argument('<oldname>', 'Organization old name')
```

Comment:
do we actually need the old name in the arguments?
- To perform that command one needs to be logged in
- If you are logged in the organization name is in the credentials

Reply:
Yes. We do. It forces the user to know what that current orgname is and what they are changing it to.

Changing orgname is important. Like changing password.

hiding the orgname in credentials , using it silently and assuming the user to be aware of it is not right.

Reply:
Do you think we should use oldname to craft the API route then?

Reply:
I don'nt want to change routing now. It is a different discussion.